<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TPD Analyser</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../tpd/tpd_analyser/resources/TPD.ico</normaloff>../tpd/tpd_analyser/resources/TPD.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1600</width>
      <height>900</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>160</red>
       <green>160</green>
       <blue>160</blue>
      </color>
     </brush>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConvert_asc_to_txt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionData_Loader">
   <property name="text">
    <string>Data Loader</string>
   </property>
  </action>
  <action name="actions">
   <property name="text">
    <string>s</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normaloff>../tpd/tpd_analyser/resources/save.png</normaloff>../tpd/tpd_analyser/resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../tpd/tpd_analyser/resources/open.png</normaloff>../tpd/tpd_analyser/resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>../tpd/tpd_analyser/resources/undo.png</normaloff>../tpd/tpd_analyser/resources/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>../tpd/tpd_analyser/resources/redo.png</normaloff>../tpd/tpd_analyser/resources/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>../tpd/tpd_analyser/resources/new.png</normaloff>../tpd/tpd_analyser/resources/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionConvert_asc_to_txt">
   <property name="icon">
    <iconset>
     <normaloff>../tpd/tpd_analyser/resources/convert.png</normaloff>../tpd/tpd_analyser/resources/convert.png</iconset>
   </property>
   <property name="text">
    <string>Convert .asc to .txt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
